package.json

```diff
@@ -58,6 +58,7 @@
     "jest": "26.1.0",
     "jsdom": "^19.0.0",
     "jsdom-global": "^3.0.2",
+    "node-gyp": "9.3.0",
```

Comment:
Fix for:
```
Usage Error: Couldn't find a script name "node-gyp" in the top-level (used by deasync@npm:0.1.21). This typically happens because some package depends on "node-gyp" to build itself, but didn't list it in their dependencies. To fix that, please run "yarn add node-gyp" into your top-level workspace. You also can open an issue on the repository of the specified package to suggest them to use an optional peer dependency.
```

I found https://github.com/vuejs/vue-jest/issues/403 but didn't go any deeper yet.